src/newgrf_industries.cpp

```diff
@@ -304,6 +304,24 @@ static uint32 GetCountAndDistanceOfClosestInstance(byte param_setID, byte layout
 			return GetCountAndDistanceOfClosestInstance(parameter, layout_filter, town_filter, this->industry);
 		}
 
+		case 0x69:
+		case 0x6A:
+		case 0x6B:
+		case 0x6C:
+		case 0x6D:
+		case 0x6E: {
+			CargoID cargo = GetCargoTranslation(parameter, this->ro.grffile);
+			int index = this->industry->GetCargoProducedIndex(cargo);
+			if (index < 0) break; // invalid cargo
+			if (variable == 0x69) return this->industry->produced_cargo_waiting[index];
+			if (variable == 0x6A) return this->industry->this_month_production[index];
+			if (variable == 0x6B) return this->industry->this_month_transported[index];
+			if (variable == 0x6C) return this->industry->last_month_production[index];
+			if (variable == 0x6D) return this->industry->last_month_transported[index];
+			if (variable == 0x6E) return Clamp(this->industry->last_cargo_accepted_at[index] - DAYS_TILL_ORIGINAL_BASE_YEAR, 0, 65535);
```

Comment:
We do not want DAYS_TILL_ORIGINAL_BASE_YEAR in new variables.
New variables should start from year 0, so just unclamped date.